S2l2a baseline update (#272)
eu-cdse/documentation: Data/Sentinel2_Baselines.qmd

---
title: "Sentinel-2 L2A Baselines"
format:
  html:
    code-fold: true
jupyter: python3
---

The Sentinel-2 L2A collection within the Copernicus Data Space Ecosystem originates from three
sources: <br />
**(1)** operational Copernicus processing, i.e. data previously accessible on the Copernicus
Open Access Hub (formerly known as Sentinels Scientific Data Hub),<br /> 
**(2)** reprocessing of the L2A

> archive since 2015 until the end of 2021 [(Collection 1)](https://sentinel.esa.int/web/sentinel/sentinel-data-access/sentinel-products/sentinel-2-data-products/collection-1-level-2a){target="_blank"}, and<br />

Reviewer: ```suggestion
archive from 2015 until the end of 2021 
```

**(3)** data generated by CloudFerro using various versions of the publicly available [Sen2Cor processor](https://step.esa.int/main/snap-supported-plugins/sen2cor/){target="_blank"}. 

These products can be differentiated by the [OData](https://documentation.dataspace.copernicus.eu/APIs/OData.html#query-by-attributes){target="_blank"} and [OpenSearch](https://catalogue.dataspace.copernicus.eu/resto/api/collections/Sentinel2/describe.xml){target="_blank"} **(origin)** attribute (ESA or CloudFerro) and by the **processingBaseline** attribute.

The current state of the archive of the Sentinel-L2A data is as follows:<br />
**29/04/2022 - now**: Baseline 509 (originating from Copernicus Open Access Hub), Baseline 99.99 (generated by CloudFerro).<br /><br />
**25/01/2022 - 06/12/2022**: Baseline 4.00 (originating from Copernicus Open Access Hub), Baseline 99.99 (generated by CloudFerro).<br /><br />
**30/03/2021 - 25/01/2022**: Baselines 3.00 3.01 (originating from Copernicus Open Access Hub), Baseline 99.99 (generate**d by CloudFerro), Baseline 5.00 (originating from the Sentinel-2 Collection 1).<br /><br />

> **04/07/2015 - 30/06/2021**: Baselines (2.01-2.13) originating from Copernicus Open Access Hub and generated by CloudFerro, Baseline 2.14 originating from Copernicus Open Access Hub, Baseline 5.00 (originating from the Sentinel-2 Collection 1)

Reviewer: 04/07/2015 - 30/06/2021** Seems like the bold text is not well replicated

> The Sentinel-2 Collection 1 (reprocessed data) featuring processing baseline 5.00 is gradually ingested into the Copernicus Data Space Ecosystem and should be complete in the Q1 2024. The products with older processing baselines should be still for at least 6 months (Q3 2024).

Reviewer: ```suggestion
The Sentinel-2 Collection 1 (reprocessed data) featuring processing baseline 5.00 is gradually ingested into the Copernicus Data Space Ecosystem and should be complete in the Q1 2024. The products with older processing baselines should still be for at least 6 months (Q3 2024).
```

---
title: "Sentinel-2 L2A Baselines"
format:
  html:
    code-fold: true
execute:
  echo: false
  output: asis
jupyter: python3

---

The Sentinel-2 L2A collection within the Copernicus Data Space Ecosystem originates from three sources: 

> **(1)** operational Copernicus processing, i.e. data previously accessible on the Copernicus Open Access Hub (formerly known as Sentinels Scientific Data Hub), 
> **(2)** reprocessing of the L2A archive from 2015 until the end of 2021 [(Collection 1)](https://sentinel.esa.int/web/sentinel/sentinel-data-access/sentinel-products/sentinel-2-data-products/collection-1-level-2a){target="_blank"}, and 
> **(3)** data generated by CloudFerro using various versions of the publicly available [Sen2Cor processor](https://step.esa.int/main/snap-supported-plugins/sen2cor/){target="_blank"}. 
> These products can be differentiated by the [OData](https://documentation.dataspace.copernicus.eu/APIs/OData.html#query-by-attributes){target="_blank"} and [OpenSearch](https://catalogue.dataspace.copernicus.eu/resto/api/collections/Sentinel2/describe.xml){target="_blank"} **(origin)** attribute (ESA or CloudFerro) and by the **processingBaseline** attribute.

Reviewer: ```suggestion
**(1)** operational Copernicus processing, i.e. data previously accessible on the Copernicus Open Access Hub (formerly known as Sentinels Scientific Data Hub);
 
**(2)** reprocessing of the L2A archive from 2015 until the end of 2021 [(Collection 1)](https://sentinel.esa.int/web/sentinel/sentinel-data-access/sentinel-products/sentinel-2-data-products/collection-1-level-2a){target="_blank"}, and ;

**(3)** data generated by CloudFerro using various versions of the publicly available [Sen2Cor processor](https://step.esa.int/main/snap-supported-plugins/sen2cor/){target="_blank"}. 

These products can be differentiated by the [OData](https://documentation.dataspace.copernicus.eu/APIs/OData.html#query-by-attributes){target="_blank"} and [OpenSearch](https://catalogue.dataspace.copernicus.eu/resto/api/collections/Sentinel2/describe.xml){target="_blank"} **(origin)** attribute (ESA or CloudFerro) and by the **'processingBaseline'** attribute.
```
